src/shared/util.js

```diff
@@ -632,19 +632,23 @@ class FeatureTest {
     if (
       (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) ||
       (typeof navigator !== "undefined" &&
-        typeof navigator?.platform === "string")
+        typeof navigator?.platform === "string" &&
+        typeof navigator?.userAgent === "string")
     ) {
       return shadow(this, "platform", {
```

Comment:
nit: maybe we could add something like `const { platform, userAgent } = navigator;`

Reply:
Sure.